Modules/Project/Resources/views/subviews/edit-resource-requirement.blade.php

```diff
@@ -0,0 +1,35 @@
+<div class="card">
+  <div id="projectResourceRequirementForm">
+    <form  action="{{ route('project.update', $project) }}" method="POST" id="updateProjectResourceRequirement" enctype="multipart/form-data">
+      @csrf
+      <input type="hidden" value="project_resource_requirement" name="update_section">
+        <div class="card-body form-body">
+          <h3 class="mb-7 font-weight-bold">Requirement Details</h3>
+            <table class="table table-bordered text-justify">
+              <h4 class="mb-3 font-weight-bold">Total Team Members Required:</h4>
+                <thead>
+                  <tr>
+                    <th>Designation</th>
+                    <th>Needed</th>
+                    <th>Deployed</th>
+                    <th>To Be Deployed</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  @foreach ($designationNames as $index => $designationName)
+                    <tr>
+                      <td><input type="text" name="designation[]" class="form-control" value="{{ ($requirements->count() > 0) ? $requirements[$index]->designation : $designationName }}" readonly></td>
+                      <td><input type="text" name="needed[]" class="form-control" value="{{ ($requirements->count() > 0) ? $requirements[$index]->total_requirement : '' }}"></td>
```

Comment:
You also need to pass slug as key for designation and instead of designationName you have to pass designation slug in needed array

```suggestion
                      <td><input type="text" name="needed[{{designationName}}]" class="form-control" value="{{ ($requirements->count() > 0) ? $requirements[$index]->total_requirement : '' }}"></td>
```

Reply:
![image](https://user-images.githubusercontent.com/92683476/218094184-9f091ff3-beef-45e9-b81b-e62634bcf562.png)
when i added `name="needed[{{$designationName}}]"`. i got the error `Undefined index: project_manager `.